### PR TITLE
fix: schema type casing

### PR DESCRIPTION
**Pull request by alexstaroselsky**
Update logic for type application casing to ensure lowercase.

### What kind of change does this PR introduce? (check at least one)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Test
 - [ ] Docs
 - [ ] Refactor
 - [ ] Build-related changes
 - [ ] Other, please describe:

### Description:

When types such as `Array<Song>` or `Song[]` are used, the `type` for the property resolves to `'Array'` rather than the officially support OpenAPI [data type](https://swagger.io/docs/specification/data-models/data-types/) of `'array'`. When the type is `'Array'`, the OpenAPI document does not correctly recognize the property as a valid array. The officially supported types are lowercase anyway:

- [string](https://swagger.io/docs/specification/data-models/data-types/#string) (this includes dates and [files](https://swagger.io/docs/specification/data-models/data-types/#file))
- [number](https://swagger.io/docs/specification/data-models/data-types/#numbers)
- [integer](https://swagger.io/docs/specification/data-models/data-types/#numbers)
- [boolean](https://swagger.io/docs/specification/data-models/data-types/#boolean)
- [array](https://swagger.io/docs/specification/data-models/data-types/#array)
- [object](https://swagger.io/docs/specification/data-models/data-types/#object)

### CODE REVIEW

**Comment by alexstaroselsky** [comment]
Found another location of `type.expression.name` in `getSchema()` will update that as well and update relevant tests to demonstrate fix.

**Comment by alexstaroselsky** [comment]
@kevinccbsg Could you please clarify who could be a good member to review this PR? Thanks.

**Review by LonelyPrincess — APPROVED** [review]

**Comment by LonelyPrincess** [comment]
> @kevinccbsg or @bri06 or @LonelyPrincess Could you please clarify who would be a good member to review this PR? Or if anything additionally is needed for it to be ready for review? Thanks.

This looks fine to me! The PR description is pretty clear, and doesn't seem like we would be missing anything to consider it complete. I've approved it, although will wait for @kevinccbsg to give the OK before merging. Thanks for your contribution! 😉

**Comment by kevinccbsg** [comment]
@all-contributors please add @alexstaroselsky for code

**Comment by alexstaroselsky** [comment]
Thanks all for helping get this through. @kevinccbsg can you confirm when this would be tentatively scheduled for release? If there are other features waiting for review/approval that would delay it, could there be a beta publish for testing?

**Comment by kevinccbsg** [comment]
@alexstaroselsky sorry for the delay Today we will do a release.

**Pull request merged**
